src/scripts/shell/star-rating/star-rating.html

```diff
@@ -0,0 +1,11 @@
+<span ng-if="$ctrl.isReadOnly">
+  <i ng-repeat="star in $ctrl.stars" class="fa fa-star read-only"
+   ng-class="{filled: star.filled}"></i>
+</span>
+<span ng-if="!$ctrl.isReadOnly">
```

Comment:
Rather than having two branches, could you just ignore clicks and hovers when readonly is set? (or use CSS to turn off user interaction maybe?)